imagej/__init__.py

```diff
@@ -682,6 +703,18 @@ def _xarray_to_dataset(self, xarr):
 
             return dataset
 
+        def _xarray_to_img(self, xarr):
+            """
+            Converts a xarray dataarray to an img, inverting C-style (slow axis first) to F-style (slow-axis last)
+            :param xarr: Pass an xarray dataarray and turn into a img.
+            :return: The img
+            """
+            if self._ends_with_channel_axis(xarr):
+                vals = np.moveaxis(xarr.values, -1, 0)
+                return self._numpy_to_img(vals) # EE: investigate here....
```

Comment:
@elevans Do you remember what you meant by "investigate here"? Does this still need validation? If so, what were/are your questions about it?

Reply:
If I remember correctly, these notes were for that 4GB limit bug and aren't related to addressing the dimension order. I'll remove them from `master`. Sorry!